app/controllers/users/unlocks_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Users::UnlocksController < Devise::UnlocksController
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.